src/xamios.tmpl.csproj

```diff
@@ -8,17 +8,18 @@
     <ProjectGuid>{E1F334C3-8F77-46C9-A28B-A8E9BAEA9FE5}</ProjectGuid>
     <ProjectTypeGuids>{FEACFBD2-3405-455C-9665-78FE426C6842};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <OutputType>Library</OutputType>
-    <RootNamespace>monotouch</RootNamespace>
-    <AssemblyName>monotouch</AssemblyName>
+    <RootNamespace></RootNamespace>
```

Comment:
So couldn't you just remove `<RootNamespace>` or it needs to be empty?

Reply:
I think VSfM likes to put it back in again if you remove it (not that it matters in this case, because it's a template VSfM won't change), so I've gotten used to do it this way.